src/EditorFeatures/CSharpTest/EditAndContinue/StatementMatchingTests.cs

```diff
@@ -825,6 +825,85 @@ public void Lambdas7()
             expected.AssertEqual(actual);
         }
 
+        [Fact]
+        public void LambdasInArrayType()
+        {
+            var src1 = "var x = new int[F(a => 1)];";
+            var src2 = "var x = new int[F(a => 2)];";
+
+            var match = GetMethodMatch(src1, src2, CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview));
+            var actual = ToMatchingPairs(match);
+
+            var expected = new MatchingPairs
+            {
+                { "var x = new int[F(a => 1)];", "var x = new int[F(a => 2)];" },
+                { "var x = new int[F(a => 1)]", "var x = new int[F(a => 2)]" },
+                { "x = new int[F(a => 1)]", "x = new int[F(a => 2)]" },
+                { "a => 1", "a => 2" }
```

Comment:
just curious, should we also expect matching pairs for `new int[...]` or `F(...)`?

Reply:
No, we don't track expressions.